src/applications/disability-benefits/all-claims/components/Autocomplete.jsx

```diff
@@ -21,22 +37,32 @@ const Autocomplete = ({
   const [results, setResults] = useState([]);
   const [activeIndex, setActiveIndex] = useState(null);
   const [ariaLiveText, setAriaLiveText] = useState('');
+  const [isScrolling, setIsScrolling] = useState(false);
 
   const containerRef = useRef(null);
   const inputRef = useRef(null);
+  const listRef = useRef(null);
   const resultsRef = useRef([]);
 
   // Delays screen reader result count reading to avoid interruption by input content reading
   const debouncedSetAriaLiveText = useRef(
-    debounce((resultCount, freeTextResult) => {
+    debounce((resultCount, inputValue) => {
       const makePlural = resultCount > 1 ? 's' : '';
 
       setAriaLiveText(
-        `${resultCount} result${makePlural}. ${freeTextResult}, (1 of ${resultCount})`,
+        `${resultCount} result${makePlural}. ${inputValue}, (1 of ${resultCount})`,
       );
     }, 700),
   ).current;
 
+  const ensureListIsVisibleOnPage = () => {
+    const labelElement = inputRef.current.shadowRoot.querySelector('label');
+
+    if (listRef.current && !isElementVisible(listRef.current)) {
+      labelElement?.scrollIntoView({ block: 'start', behavior: 'smooth' });
```

Comment:
We'll want to respect the user's `prefers-reduced-motion` settings so let's do a check and set the behavior to `instant` in that case

```
window.matchMedia(
    '(prefers-reduced-motion: reduce)',
  ).matches;
```